### DISCUSSION TITLE
Can't find Unknown Sp.

### DISCUSSION THREAD

**Discussion by Vollpflock** [Mattk70/Chirpity-Electron#110] [Q&A] [answered]
Where can I find the Unknown Sp.? 

![Screenshot 2024-04-14 at 09 31 15](https://github.com/Mattk70/Chirpity-Electron/assets/26574621/71655204-675f-4c98-902f-7e28a5dab296)
![Screenshot 2024-04-14 at 09 31 35](https://github.com/Mattk70/Chirpity-Electron/assets/26574621/e0e9acc1-041d-4fe9-9f00-8f88be9c7349)
![Screenshot 2024-04-14 at 09 31 46](https://github.com/Mattk70/Chirpity-Electron/assets/26574621/0c02cd11-e484-4bee-9adf-0c055c4dc26d)
![Screenshot 2024-04-14 at 09 32 05](https://github.com/Mattk70/Chirpity-Electron/assets/26574621/de9d8784-fb13-478c-a026-efff4f243fc8)
![Screenshot 2024-04-14 at 09 32 20](https://github.com/Mattk70/Chirpity-Electron/assets/26574621/630b4d27-3bdd-437d-91e3-b2946076882f)

EDIT1:

In Archive, Unknown Sp. only shows up if I select a date and not all of them show up. I'm positive that I edited more records with Unknown Sp.

![Screenshot 2024-04-14 at 10 05 00](https://github.com/Mattk70/Chirpity-Electron/assets/26574621/f12f90c3-cbac-4064-a575-047dc83ac584)
![Screenshot 2024-04-14 at 10 04 48](https://github.com/Mattk70/Chirpity-Electron/assets/26574621/d1a41e39-2b5d-4540-8cfe-73354b71c0b3)

EDIT 2:
I can find them when I choose a date and no location. So after editing the records to "Unknown Sp.", I should've amended the location again? 
![Screenshot 2024-04-14 at 10 22 25](https://github.com/Mattk70/Chirpity-Electron/assets/26574621/42520097-6fd4-4ec4-b40a-a0dc09ed60be)

**Comment by Mattk70** [comment, marked as answer]
I think the issue here is that Unknown Sp. is excluded by your list. It's not returned as a "local species" for your area. If you click the icon until you see the loudspeaker, you should have no problem retrieving all your Unknown  Sp. records.

And, whilst this is the answer for versions up to 1.6.9, it isn't an intuitive thing. I'll change the lists so "Unknown Sp." is included in all lists from version 1.6.10.
